freqtrade/templates/FreqaiExampleStrategy.py

```diff
@@ -39,7 +39,7 @@ class FreqaiExampleStrategy(IStrategy):
     use_exit_signal = True
     # this is the maximum period fed to talib (timeframe independent)
     startup_candle_count: int = 40
-    can_short = False
+    can_short = True
```

Comment:
I think the idea to have this as "false" was to allow the default sample strategy to also work on spot markets (freqtrade will not allow you to load a short strategy in spot markets).

As such - i'm not entirely sure if we should really be changing this to True - as it'll automatically exclude all spot markets.

Reply:
In principle I'm fine with that - just pointing out _why_ we set this to False in the first place :) 
requiring long/short support will however be problematic for quite a lot of people (US, europe) - as bigger exchanges started to lock us out from their futures markets.

it's anyway pointed at a non-develop branch but wagner's  - so will probably experience another review together with that branch.

Reply:
I agree with you guys. I vote for setting it to True, but I understand Matthias concerns and I think we can easily add a spot strategy freqAI example (or add a comment inside the current strategy giving some orientation).

Reply:
works for me :+1: